tools/scan-build-py/libscanbuild/analyze.py

```diff
@@ -639,7 +639,8 @@ def dispatch_ctu(opts, continuation=run_analyzer):
                 + opts['flags'] + [opts['file']]
             triarch = get_triple_arch(cmd, cwd)
             ctu_options = ['ctu-dir=' + os.path.join(ctu_config.dir, triarch),
-                           'reanalyze-ctu-visited=true']
```

Comment:
Do we need this reanalyze flag at all? As far as I remember we no longer have this.

Reply:
I think it was removed, so we don't.